x/pairing/client/cli/query_provider_epoch_cu.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/lavanet/lava/x/pairing/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdProviderEpochCu() *cobra.Command {
```

Comment:
change to CmdProvidersEpochCu

Reply:
done. see https://github.com/lavanet/lava/commit/3e4937ad2fc531aae9d346db710b69820845ff4e